central/cluster/datastore/datastore_impl.go

```diff
@@ -152,6 +155,24 @@ func (ds *datastoreImpl) UpdateClusterStatus(ctx context.Context, id string, sta
 	return ds.clusterStorage.Upsert(ctx, cluster)
 }
 
+func (ds *datastoreImpl) removeImageIntegrationByCluster(ctx context.Context, cluster *storage.Cluster) {
```

Comment:
nit: Let's try to keep it consistent with the other related functions: `removeClusterImageIntegrations`

Reply:
Also maybe move this directly below `postRemoveCluster` with the rest of the related functions